### PR TITLE
coprocessor: extract a utility function 'is_unsigned'

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <heng@lonng.org>

## What have you changed? (mandatory)

Extract a common utility function `is_unsigned`.

## What are the type of the changes? (mandatory)

The currently defined types are listed below, please pick one of the types for this PR by removing the others:
- Engineering (engineering change which doesn't change any feature or fix any issue)

## How has this PR been tested? (mandatory)

Exists unit tests.

### CODE REVIEW

File: src/coprocessor/dag/rpn_expr/impl_cast.rs [hunk 1]
```diff
@@ -55,6 +47,11 @@ pub fn get_cast_fn_rpn_node(
     })
 }
 
+#[inline]
+fn is_unsigned(ft: &FieldType) -> bool {
```
Comment (breeswish) [inline src/coprocessor/dag/rpn_expr/impl_cast.rs]: why not a FieldType method?

Comment (lonng) [reply src/coprocessor/dag/rpn_expr/impl_cast.rs]: The FieldType is protobuf type, we cannot attach a method to it.

Comment (breeswish) [reply src/coprocessor/dag/rpn_expr/impl_cast.rs]: Then how about in `FieldTypeAccessor`?

Comment (lonng) [reply src/coprocessor/dag/rpn_expr/impl_cast.rs]: @breeswish https://github.com/tikv/tikv/pull/5095

**Pull request merged**